cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/CruiseControlRebalanceResponse.java

```diff
@@ -19,6 +20,8 @@ public class CruiseControlRebalanceResponse extends CruiseControlResponse {
         // Proposal is not in progress unless response from Cruise Control says otherwise
         // Sourced from the "progress" field in the response with value "proposalStillCalaculating"
         this.isProposalStillCalaculating = false;
```

Comment:
@ppatierno Hi, not any change by you but seems like there is a typo here `isProposalStillCalaculating.

Reply:
Yeah thanks. I will leave the regression passing and then making the change before merging (it won't have impact on regression again).

Reply:
Damn merged the PR before fixing this, going to open a trivial new one.